### PR TITLE
Some small improvements to 1.12.2

### PR DESCRIPTION
**Pull request by HowardZHY**
acceptableRemoteVersions = "*", clientSideOnly=true

Remove spam debug msg

### CODE REVIEW

File: viaforge-mc1122/src/main/resources/mcmod.info [hunk 1]
```diff
@@ -4,15 +4,12 @@
     "name": "ViaForge",
     "description": "Client-side Implementation of the Via* projects for Minecraft Forge",
     "version": "${version}",
-    "mcversion": "[1.12.2]",
+    "mcversion": "[1.12,)",
     "url": "https://github.com/ViaVersion/ViaForge",
-    "updateUrl": "",
```
Comment (FlorianMichael) [inline viaforge-mc1122/src/main/resources/mcmod.info:9]: maybe we can fill this with some correct value?

File: viaforge-mc1122/src/main/resources/mcmod.info [hunk 2]
```diff
@@ -4,15 +4,12 @@
     "name": "ViaForge",
     "description": "Client-side Implementation of the Via* projects for Minecraft Forge",
     "version": "${version}",
-    "mcversion": "[1.12.2]",
+    "mcversion": "[1.12,)",
```
Comment (FlorianMichael) [inline viaforge-mc1122/src/main/resources/mcmod.info:7]: Did you try if loading on 1.12.0 and 1.12.1 Forge works?

**Review by FlorianMichael — CHANGES_REQUESTED** [review]
Minor things

File: viaforge-mc1122/src/main/resources/mcmod.info (hunk 1 above)
Comment (HowardZHY) [reply viaforge-mc1122/src/main/resources/mcmod.info:9]: just delete it, it's almost useless for end users

File: viaforge-mc1122/src/main/resources/mcmod.info (hunk 2 above)
Comment (HowardZHY) [reply viaforge-mc1122/src/main/resources/mcmod.info:7]: yes

**Review by FlorianMichael — APPROVED** [review]
Thank you! Can you also pr to the 1.8 branch?

**Pull request merged**